revolv/templates/base/partials/foot.html

```diff
@@ -51,15 +51,15 @@
           people and communities to invest collectively in renewable energy.
           All donations are tax-deductible.
         </p>
-		<dl>
-			<dd><a href="https://old.re-volv.org/">Visit our old website</a></dd>
-		</dl>
-		<dl>
-			<dt><a href="http://goo.gl/forms/S1TEKTNgNz">How do you like our new website?</a></dt>
-		</dl>
-		<p>
-			Take a short survey and let us know. We would love to hear your feedback!
-		</p>
+		    <dl>
```

Comment:
Hi @click-ripple, when you say "These contents are added because there is no confirmation for this part of the implementation", do you mean about the `<dl>` changes, or the "hard-coded" text? This comment thread pertains to only the `<dl>` changes, please use the comment button all the way at the bottom of the page to discuss issues not related to specific lines of code.

Thank you so much for fixing up the indentation! Looking forward to it!